src/js/scriptlets/dom-inspector.js

```diff
@@ -800,8 +800,7 @@ var start = function() {
             document.removeEventListener(ev.type, onReady);
         }
         vAPI.messaging.sendTo(loggerConnectionId, domLayout.get());
-        vAPI.domFilterer.toggle(false);
-        highlightElements();
+        vAPI.domFilterer.toggle(false, highlightElements);
```

Comment:
Ok, I think I see what was happening: the highlighter was called before the user stylesheets were removed (from extension process), hence causing the highlighter to see only empty boxes. I was puzzle as to why there was this glitch with Firefox but deferred investigation. Nice catch.